### ISSUE TITLE
Consuming asset fails

### ISSUE THREAD

**Issue by jernejpregelj** [oceanprotocol/squid-js#114]
## Current Behavior

Running src/examples/InitializeAgreement.ts fails with "Error invalid bytes32 value"

## Step to Reproduce the problem

Please provide detailed steps for reproducing the issue.

1. Uncomment error logging on https://github.com/oceanprotocol/squid-js/blob/69ed0428b175aaa7e821b4bee3c577fb6ba43f09/src/keeper/Event.ts#L55
2. npm run run src/examples/InitializeAgreement.ts 

## Specifications

* Squid-js: develop (69ed042)

## Failure Logs

```
Jernejs-MacBook-Pro:squid-fresh jernejpregelj$ npm run run src/examples/InitializeAgreement.ts 

> @oceanprotocol/squid@0.2.7 run /Users/jernejpregelj/oceanprotocol/squid-fresh
> ts-node "src/examples/InitializeAgreement.ts"

Loaded OceanMarket from spree
Loaded OceanAuth from spree
Loaded OceanToken from spree
Loaded ServiceAgreement from spree
Loaded AccessConditions from spree
Loaded PaymentConditions from spree
Loaded DIDRegistry from spree
Registered asset with did: did:op:f304925d0a5e40d7b79f3472e1092b93e2c2889d84f54fd985ac09a7352de071
ServiceAgreement Id: eb160e338bf948a0b9cba46765e40df093a20b8c4ae145438c2b29dd59591422
ServiceAgreement Signature: 0xcbc8e10f3bdba2a635180f0ee93ae87fdfa8c349c77476956b763f2f840e0bb86d692b61160462231fd916f313402d2a07e278b725ec8d21547d43a8ac4142b01c
err { Error: invalid bytes32 value (arg="", coderType="bytes32", value="eb160e338bf948a0b9cba46765e40df093a20b8c4ae145438c2b29dd59591422")
    at Object.throwError (/Users/jernejpregelj/oceanprotocol/squid-fresh/node_modules/web3-eth-abi/node_modules/ethers/utils/errors.js:68:17)
    at CoderFixedBytes.encode (/Users/jernejpregelj/oceanprotocol/squid-fresh/node_modules/web3-eth-abi/node_modules/ethers/utils/abi-coder.js:433:20)
    at /Users/jernejpregelj/oceanprotocol/squid-fresh/node_modules/web3-eth-abi/node_modules/ethers/utils/abi-coder.js:605:59
    at Array.forEach (<anonymous>)
    at pack (/Users/jernejpregelj/oceanprotocol/squid-fresh/node_modules/web3-eth-abi/node_modules/ethers/utils/abi-coder.js:604:12)
    at CoderTuple.encode (/Users/jernejpregelj/oceanprotocol/squid-fresh/node_modules/web3-eth-abi/node_modules/ethers/utils/abi-coder.js:764:16)
    at AbiCoder.encode (/Users/jernejpregelj/oceanprotocol/squid-fresh/node_modules/web3-eth-abi/node_modules/ethers/utils/abi-coder.js:897:77)
    at ABICoder.encodeParameters (/Users/jernejpregelj/oceanprotocol/squid-fresh/node_modules/web3-eth-abi/src/index.js:96:27)
    at ABICoder.encodeParameter (/Users/jernejpregelj/oceanprotocol/squid-fresh/node_modules/web3-eth-abi/src/index.js:84:17)
    at /Users/jernejpregelj/oceanprotocol/squid-fresh/node_modules/web3-eth-contract/src/index.js:333:28
    at Array.map (<anonymous>)
    at Contract._encodeEventABI (/Users/jernejpregelj/oceanprotocol/squid-fresh/node_modules/web3-eth-contract/src/index.js:319:16)
    at Contract._generateEventOptions (/Users/jernejpregelj/oceanprotocol/squid-fresh/node_modules/web3-eth-contract/src/index.js:548:22)
    at Contract.getPastEvents (/Users/jernejpregelj/oceanprotocol/squid-fresh/node_modules/web3-eth-contract/src/index.js:655:49)
    at GenericContract.<anonymous> (/Users/jernejpregelj/oceanprotocol/squid-fresh/src/keeper/contracts/ContractBase.ts:22:30)
    at step (/Users/jernejpregelj/oceanprotocol/squid-fresh/src/keeper/contracts/ContractBase.ts:32:23)
    at Object.next (/Users/jernejpregelj/oceanprotocol/squid-fresh/src/keeper/contracts/ContractBase.ts:13:53)
    at /Users/jernejpregelj/oceanprotocol/squid-fresh/src/keeper/contracts/ContractBase.ts:7:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/jernejpregelj/oceanprotocol/squid-fresh/src/keeper/contracts/ContractBase.ts:3:12)
    at GenericContract.ContractBase.getEventData (/Users/jernejpregelj/oceanprotocol/squid-fresh/src/keeper/contracts/ContractBase.ts:46:16)
    at Event.<anonymous> (/Users/jernejpregelj/oceanprotocol/squid-fresh/src/keeper/Event.ts:44:43)
    at step (/Users/jernejpregelj/oceanprotocol/squid-fresh/src/keeper/Event.ts:32:23)
    at Object.next (/Users/jernejpregelj/oceanprotocol/squid-fresh/src/keeper/Event.ts:13:53)
    at fulfilled (/Users/jernejpregelj/oceanprotocol/squid-fresh/src/keeper/Event.ts:4:58)
    at process.internalTickCallback (internal/process/next_tick.js:77:7)
  reason: 'invalid bytes32 value',
  code: 'INVALID_ARGUMENT',
  arg: '',
  coderType: 'bytes32',
  value:
   'eb160e338bf948a0b9cba46765e40df093a20b8c4ae145438c2b29dd59591422' }
```
Error is repeated multiple times.

**Comment by Pedro-vk** [comment]
@jernejpregelj it should be fixed on version `0.2.8`. If you think that is all ok, close the issue :) Thanks!

**Comment by jernejpregelj** [comment]
Just tested it and it is fixed
